src/utils/sendError.ts

```diff
@@ -0,0 +1,9 @@
+export function sendErrorMessage(
+  errorText: string,
+  setErrorMessage: React.Dispatch<React.SetStateAction<string>>,
+) {
+  setErrorMessage(errorText);
+  setTimeout(() => {
+    setErrorMessage('');
+  }, 3000);
+}
```

Comment:
This logic can be moved to Error component as logic in useEffect hook. You don't need to move it in utils